server/games/game.py

```diff
@@ -703,6 +710,17 @@ async def update_game_stats(self):
                 )
             )
 
+            if self.matchmaker_queue_name is not None:
+                queue_id_select = select([matchmaker_queue.c.id]).where(
+                    matchmaker_queue.c.technical_name == self.matchmaker_queue_name
+                )
```

Comment:
Why not take a `MatchmakerQueue` object instead of just the name, then you will already have access to the id?

Reply:
The object doesn't store the id currently. We can add and either pass it or the id only, but right now there is still the hardcoded `ladder1v1` fallback queue, which wouldn't have a database id attached:
https://github.com/FAForever/server/blob/1ec2a7bd145ea4360ed38f76ea5705560098472e/server/ladder_service.py#L54-L64

We could of course make the id field optional and give this one a `None` id if you think that is the better way.

Reply:
We don't need the fallback anymore, we can remove it. It's already in progress here #623, but hasn't been quite finished.

Reply:
Maybe I'll take over #623 then, since it's been some time? If that's alright with @RemcoSchrijver.
Then afterwards I'll add the id field.

Reply:
Yea that's a good idea. You can just pull his commits into a new branch and finish deleting the related fallback stuff.

Reply:
Added the id, now depends on #656